roles/splunk/tasks/adhoc_clean_kvstore.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: Stop Splunkd service
+  include_tasks: splunk_stop.yml
+
+- name: Clean KVStore
+  ansible.builtin.command: "{{ splunk_home }}/bin/splunk -auth {{ splunk_auth }} clean kvstore --local --answer-yes"
```

Comment:
Cleaning the KV Store does not require authentication, since splunk is stopped. If you don't mind, you can add me as a collaborator to your fork, and I can add a task to login to splunk once, so we can avoid all this login junk, which has been getting on my last nerve. Requiring `no_log` inclusion for these tasks makes it difficult to troubleshoot when things don't work.

Reply:
Done, I just added you. I would be very interested in a Splunk Login task in a separate file so that it can be called as needed...

Reply:
Pushed. I have not tested it yet, so I may have overlooked something. But you can review and test it if you want to, and let me know if I missed something.

I can test it, but I'll need to spin up a few environments to do so, and it can take some time.